pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -108,6 +108,8 @@ var CapabilitiesByProvisionerKey = map[string][]StorageCapabilities{
 	// Infinidat
 	"infinibox-csi-driver/iscsiorfibrechannel": {{rwx, block}, {rwo, block}, {rwo, file}},
 	"infinibox-csi-driver/nfs":                 {{rwx, file}, {rwo, file}},
+	// vSphere
+	"csi.vsphere.vmware.com": {{rwo, block}, {rwo, file}},
```

Comment:
I think you should add {rwx, file}, although I don't see Parameters we can check rely on. Anyway, the user can always force the DV to use rwo if rwx is not supported.

Reply:
Added